src/main/java/jalasoft/com/models/TaskPage.java

```diff
@@ -0,0 +1,13 @@
+package jalasoft.com.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskPage extends Page<Task> {
+
+	public String name = "task page";

```

Comment:
Why public?

Reply:
this should be a constant

Reply:
I deleted this class since I was just making some tests with this kind of class. Instead, I am using the generic class Page<T> to handle a list of tasks.